src/apps/chifra/internal/receipts/output.go

```diff
@@ -45,12 +65,77 @@ func (opts *ReceiptsOptions) ReceiptsInternal() (err error, handled bool) {
 	if opts.Globals.ApiMode {
 		return nil, false
 	}
+	if opts.Articulate {
+		err = opts.Globals.PassItOn("getReceipts", opts.Globals.Chain, opts.toCmdLine(), opts.getEnvStr())
+		return err, true
+	}
 
-	handled = true
-	err = opts.Globals.PassItOn("getReceipts", opts.Globals.Chain, opts.toCmdLine(), opts.getEnvStr())
-	// EXISTING_CODE
+	notFound := make([]error, 0)
+	defer func() {
+		for _, err := range notFound {
+			fmt.Fprintln(os.Stderr, err)
+		}
+	}()
+	clientVersion, err := rpcClient.GetVersion(opts.Globals.Chain)
+	if err != nil {
+		return err, true
+	}
+	erigonUsed := utils.IsClientErigon(clientVersion)
 
-	return
+	getTransaction := func(models chan types.Modeler[types.RawReceipt], errors chan error) {
+		for idIndex, rng := range opts.TransactionIds {
```

Comment:
Add a TODO item above this line that says "// TODO: stream transaction identifiers